### PR TITLE
Fix linear regression training setting

### PR DESCRIPTION
**Pull request by jan-www**
Fix linear regression parameters. SGD will iterate 10000 times at most to ensure convergence

### CODE REVIEW

File: test/browser/generic/tabular.js [hunk 1]
```diff
@@ -4,66 +4,66 @@ const tf = require('@tensorflow/tfjs-core');
 describe('Encodings', function () {
   describe('LabelEncoder', function () {
     it('test the label encoding on array', async function () {
-      const data = [ 'dog', 'cat', 'man', 'dog', 'cat', 'man', 'man', 'cat' ];
-      const encode = new datacook.Encoder.LabelEncoder(data);
-      const fit_data = [
+      let data = ['dog', 'cat', 'man', 'dog', 'cat', 'man', 'man', 'cat'];
+      let encode = new datacook.Encoder.LabelEncoder(data);
+      let fit_data = [
```
Comment (yorkie) [inline test/browser/generic/tabular.js:9]: The above should be constants, why does this change?

**Review by yorkie — CHANGES_REQUESTED** [review]
And the file "test/node/stat/hist.ts" should be removed, too.

File: test/browser/generic/tabular.js (hunk 1 above)
Comment (jan-www) [reply test/browser/generic/tabular.js:9]: probabiliy a version issue

**Comment by yorkie** [comment]
There are conflicts at the package-lock.json file.

**Pull request merged**